doc/input-output-reference/src/overview/group-zone-forced-air-units.tex

```diff
@@ -480,26 +480,36 @@ \subsubsection{Outputs}\label{outputs-043}
 
 \paragraph{Zone Ideal Loads Outdoor Air Standard Density Volume Flow Rate {[}m3/s{]}}\label{zone-ideal-loads-outdoor-air-standard-density-volume-flow-rate-m3s}
 
-The volume flow rate of the outdoor air stream in m3/s using the standard density. The standard density is determined for dry air at the standard barometric pressure for the location's elevation and a temperature of 20.0ºC. The standard density does not vary over time.
+The volume flow rate of the outdoor air stream in m3/s using the standard density. The standard density is determined for dry air at the standard barometric pressure for the location's elevation and a temperature of 20.0º\si{C}. The standard density does not vary over time.
 
 \paragraph{Zone Ideal Loads Supply Air Mass Flow Rate {[}kg/s{]}}\label{zone-ideal-loads-supply-air-mass-flow-rate-kgs}
 
 The mass flow rate of the supply air stream in kg/s.
 
 \paragraph{Zone Ideal Loads Supply Air Standard Density Volume Flow Rate {[}m3/s{]}}\label{zone-ideal-loads-supply-air-standard-density-volume-flow-rate-m3s}
 
-The volume flow rate of the supply air stream in m3/s using the standard density. The standard density is determined for dry air at the standard barometric pressure for the location's elevation and a temperature of 20.0ºC. The standard density does not vary over time.
+The volume flow rate of the supply air stream in m3/s using the standard density. The standard density is determined for dry air at the standard barometric pressure for the location's elevation and a temperature of 20.0º\si{C}. The standard density does not vary over time.
 
-\paragraph{Zone Ideal Loads Supply Air Temperature {[}C{]}}\label{zone-ideal-loads-supply-air-temperature-c}
+\paragraph{Zone Ideal Loads Supply Air Temperature {[}\si{C}{]}}\label{zone-ideal-loads-supply-air-temperature-c}
 
-The dry bulb temperature of the supply air stream in ºC.
+The dry bulb temperature of the supply air stream in º\si{C}.
 
 
 \paragraph{Zone Ideal Loads Supply Air Humidity Ratio {[}kgWater/kgDryAir{]}}\label{zone-ideal-loads-supply-air-humidity-ratio-kgWaterkgDryAir}
 
 The humidity ratio of the supply air stream in kgWater/kgDryAir.
 
 
+\paragraph{Zone Ideal Loads Mixed Air Temperature {[}\si{C}{]}}\label{zone-ideal-loads-supply-air-temperature-c}
+
+The dry bulb temperature of the mixed air stream after mixing the recirculation air stream and outdoor air stream (if present), in º\si{C}.
+
+
+\paragraph{Zone Ideal Loads Mixed Air Humidity Ratio {[}kgWater/kgDryAir{]}}\label{zone-ideal-loads-supply-air-humidity-ratio-kgWaterkgDryAir}
+
+The humidity ratio of the mixed air stream after mixing the recirculation air stream and outdoor air stream (if present) in kgWater/kgDryAir.
```

Comment:
New addition. Does it look ok?

Reply:
Yes, this looks reasonable. I thought about adding "entering the zone inlet node" to be more specific but with the change for zone/spaces coming it's probably better at this point to leave this generic.